Slight refactor + Ignore files and directories (#643)
domluna/JuliaFormatter.jl: README.md

end
```

### `ignored`

An array of paths to files and directories (with possible Glob wildcards)

> which should not be processed.

Reviewer: should be under File Options no?

> ### `ignored`
> 
> An array of paths to files and directories (with possible Glob wildcards)
> which should not be processed.

Reviewer: ```suggestion
### `ignored`

An array of paths to files and directories (with possible Glob wildcards)
which will not be formatted.
```
